### PR TITLE
[REF] GeomUtils.cpp

### PR DESCRIPTION
**Pull request by SamuilsRulovs**
* Added exceptions to GeomUtils.cpp

### CODE REVIEW

**Review by pjanck — CHANGES_REQUESTED** [review]
Good proposals. The Inconsistent..Exceptions take `string` only as input as well. Since you do not have `EXPRESSReferences` here, you should populate the exceptions with only string messages.

File: Core/src/IfcGeometryConverter/GeomUtils.cpp [hunk 1]
```diff
@@ -130,23 +130,23 @@ void GeomUtils::extrude(
...
 		*/
 	}
 	if( poly_data->points.size() > 0 )
 	{
 		std::cout << "extrude: points vec should be empty" << std::endl;
 		return;
 		/*
-		throw oip::InconsistentModellingException(poly_data, "points vec should be empty");
+		throw oip::InconsistentModellingException("points vec should be empty");
 		*/
```
Comment (pjanck) [inline Core/src/IfcGeometryConverter/GeomUtils.cpp]: All throwing code is still commented out, and the legacy code wasn't replaced.

**Review by pjanck — CHANGES_REQUESTED** [review]
Apply similar changes to all parts, as noted with one. Replace legacy `std::cout` printing with `throw`s.

File: Core/src/IfcGeometryConverter/GeomUtils.cpp (hunk 1 above)
Comment (SamuilsRulovs) [reply Core/src/IfcGeometryConverter/GeomUtils.cpp]: Resolved: 3353249

File: Core/src/IfcGeometryConverter/GeomUtils.cpp [hunk 2]
```diff
@@ -127,27 +127,15 @@ void GeomUtils::extrude(
 {
 	if( face_loops_input.size() == 0 )
 	{
-		std::cout << "extrude: face_loops_input.size() == 0" << std::endl;
-		return;
-		/*
-		throw oip::InconsistentModellingException("face_loops_input.size() == 0");
-		*/
+		throw oip::InconsistentModellingException("face_loops_input.size() == 0");
```
Comment (pjanck) [inline Core/src/IfcGeometryConverter/GeomUtils.cpp]: Please include the name of the function in the messages (similarly to how it was):
```suggestion
		throw oip::InconsistentModellingException("GeomUtils::extrude(): face_loops_input.size() == 0");	
```

Comment (pjanck) [reply Core/src/IfcGeometryConverter/GeomUtils.cpp]: Also, similarly to other messages, write out, that `face_loops_input.size() should not be 0`.

**Review by pjanck — CHANGES_REQUESTED** [review]
Please apply the comment for one line at other lines as well.

File: Core/src/IfcGeometryConverter/GeomUtils.cpp (hunk 2 above)
Comment (SamuilsRulovs) [reply Core/src/IfcGeometryConverter/GeomUtils.cpp]: Resolved:  8eb52be

**Review by pjanck — APPROVED** [review]

**Review by pjanck — APPROVED** [review]

**Pull request merged**